### PR TITLE
docs: update dialtone usage documentation

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description

Updated dialtone usage documentation regarding the need of adding `dialtone-theme-light` or `dialtone-theme-dark` to the <body> in order for dialtone's CSS variables to work properly.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

**Review by iropolo — APPROVED** [review]
🚀

File: README.md [hunk 1]
```diff
@@ -15,6 +15,23 @@ Once installed, add the following line in your CSS/LESS file:
 @import "node_modules/@dialpad/dialtone/lib/dist/css/dialtone.css";
 ```
 
+And dialtone's theme class to the `<body>`
```
Comment (iropolo) [inline README.md]: Should be `Add`? Or I'm not understanding the sentence correctly

Comment (braddialpad) [reply README.md]: Yeah should be add

File: README.md [hunk 2]
```diff
@@ -15,6 +15,23 @@ Once installed, add the following line in your CSS/LESS file:
...
+
+**Dark mode**
+```html
+<body class="dialtone-theme-dark">...</body>
+```
+
+It is important to add either of those classes in order to make dialtone work, as CSS variables
+definition change according to the selected mode (light, dark). 
+
+If the class is not added, the CSS variables aren't going to be defined.
```
Comment (braddialpad) [inline README.md]: Can probably simplify this a bit, it makes sense it's just a bit wordy. Maybe something like:

"This will define the Dialtone variables for your desired theme. It is required to do this for Dialtone to function."

File: README.md (hunk 1 above)
Comment (juliodialpad) [reply README.md]: My intention was continuing the previous sentence, but makes sense to change it to "Add"

Comment (braddialpad) [reply README.md]: Oh I see that now haha. You couldn't see the previous line in the PR so it seemed like it didn't make sense.

**Comment by juliodialpad** [comment]
Changed the documentation a bit again, please re-review

**Review by braddialpad — APPROVED** [review]
Looks good thanks.
